pkg/apis/rsyslog/types_rsyslog.go

```diff
@@ -62,6 +65,16 @@ type LoggingRule struct {
 	Severity int
 }
 
+// AuditConfig contains options to configure the audit system.
+type AuditConfig struct {
+	// Enabled determines whether auditing configurations are applied to the nodes or not.
+	// Defaults to true.
+	Enabled *bool
```

Comment:
Yes, I think that makes sense. For some reason I thought that I wouldn't be able to default it to true, but you're right that that isn't a problem since `AuditConfig` is optional (can be nil).

Reply:
Actually, it might be better flip it and rename the field to `Disabled` - this way users have to explicitly specify that they do not want audit rules and hence skip on this security feature. @ialidzhikov wdyt?

Reply:
As we talked, let's keep it enable, if possible. `disabled: false` is a double negation which one idea harder to follow.